etcdctl/ctlv2/command/error.go

```diff
@@ -30,10 +32,15 @@ const (
 	ExitClusterNotHealthy
 )
 
-func handleError(code int, err error) {
-	fmt.Fprintln(os.Stderr, "Error: ", err)
-	if cerr, ok := err.(*client.ClusterError); ok {
-		fmt.Fprintln(os.Stderr, cerr.Detail())
+func handleError(code int, err error, isJSON bool) {
```

Comment:
change this to handleError(c c *cli.Context, exitCode int, err error)

check if we need json format inside handleError. current code is too messy.